src/hammer-tech/stackup.py

```diff
@@ -279,6 +279,7 @@ def get_width_spacing_start_twwt(self, tracks: int, force_even: bool = False) ->
         """
         widths_and_spacings = self.power_strap_widths_and_spacings
         spacing = widths_and_spacings[0].min_spacing
+        assert self.pitch - self.min_width == spacing, "Tech plugin is malformed for metal {}, the minimum spacing in the width-spacing list must be the same as (pitch - min_width).".format(self.name)
```

Comment:
Minor point, can we add unit tests for these kinds of bad cases?

Reply:
IDK what you mean- there are no unit tests in the tech plugin.

Reply:
Well right now they live in `hammer-vlsi/tech_test.py` IIRC

Reply:
but this isn't a problem with the tech code- it's catching a problem with a bad plugin. Are you saying you want a test that contains a bad plugin?

Reply:
Yeah, a test that purposely feeds it a bad plugin and check that it's caught

Reply:
I think that's beyond the scope of this PR, but we can file an issue.

Reply:
Filed #402